website/docs/installation/documentdb/kubernetes.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 3
+---
+
+# Kubernetes
+
+FerretDB uses PostgreSQL with [DocumentDB extension](https://github.com/microsoft/documentdb) as the database engine.
+
+Ensure to have a running Kubernetes cluster before proceeding with the installation.
+
+You can deploy PostgreSQL with DocumentDB extension using any of our provided images.
+Please see the [Docker installation docs](../documentdb/docker.md) to learn more about the available images.
+
+:::tip
+We strongly recommend specifying the full image tag (e.g., `17-0.103.0-ferretdb-2.2.0`)
+to ensure consistency across deployments.
+For more information on the best FerretDB image to use, see the [DocumentDB release notes](https://github.com/FerretDB/documentdb/releases/).
```

Comment:
_the_ DocumentDB releaes notes?

Reply:
I don't quite understand this comment. Do you mean I should remove "the" or that those are not release notes?

Reply:
Those are release notes – for all versions of DocumentDB. It links to releases page, not to a particular release page. Should it be "_the_ DocumentDB release notes"?

Reply:
Yes. It has been pluralized with "notes" which covers all versions of DocumentDB.